fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/MetricInstance.java

```diff
@@ -94,6 +95,18 @@ public MetricInstance(LogicalMetricInfo logicalMetricInfo, MetricMaker maker, Fi
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Construct a MetricInstance from FieldNames.
+     *
+     * @param logicalMetricInfo  Logical metric info provider
+     * @param maker  The Metric Maker that creates the actual Logical Metric
+     */
+    public MetricInstance(LogicalMetricInfo logicalMetricInfo, MetricMaker maker) {
```

Comment:
This is needed for https://github.com/yahoo/fili/pull/610/files#diff-bd8bdd53f38556177cbc9e9b6b60d765R72 in `WikiMetricLoader.java`